src/app/log.hpp

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
```

Comment:
Should we set 2024 year to all new files?
```suggestion
 * Copyright (C) 2024 Renesas Electronics Corporation.
```

Reply:
done